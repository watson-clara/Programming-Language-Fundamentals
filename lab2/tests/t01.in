debug off
stopwords on
load tracks.txt
preprocess
mcw love
mcw you
mcw happy
mcw hello
mcw word
mcw ain't
mcw nobody
mcw got
mcw time
mcw for
mcw that
mcw to
mcw be
mcw or
mcw not
mcw 2
mcw bee
mcw never
mcw going
mcw to
mcw give
mcw you
mcw up
mcw never
mcw gonna
mcw let
mcw u
mcw down
mcw were
mcw you
mcw expecting
mcw something
mcw else
mcw nunca
mcw voy
mcw a
mcw corretear
mcw ni
mcw abandonare
mcw goodbye
