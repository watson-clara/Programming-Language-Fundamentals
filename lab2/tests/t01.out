you
are
birthday
hello
up
no
but
it
is

i

your

alone
b
bee
be
back

me
are
my
be
be
me
got
my
you
are
heroes
special
i
mas
conquistarte


bu

my
